src/There4/Slim/Test/WebTestClient.php

```diff
@@ -10,26 +10,46 @@ class WebTestClient
     public $request;
     public $response;
 
-    // We support these methods for testing. These are available via
-    // `this->get()` and `$this->post()`. This is accomplished with the
-    // `__call()` magic method below.
-    public $testingMethods = array('get', 'post', 'patch', 'put', 'delete', 'head');
-
     public function __construct(Slim\Slim $slim)
     {
         $this->app = $slim;
     }
 
-    // Implement our `get`, `post`, and other http operations
     public function __call($method, $arguments)
     {
-        if (in_array($method, $this->testingMethods)) {
-            list($path, $data, $headers) = array_pad($arguments, 3, array());
-            return $this->request($method, $path, $data, $headers);
-        }
         throw new \BadMethodCallException(strtoupper($method) . ' is not supported');
```

Comment:
Unsure if this line should be kept, or removing the entire __call method and the test asserting this exception.